packages/web-components/examples/codesandbox/components-react/content-item/src/index.css

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+html,
+body {
+  margin: 0;
+}
+
+body {
+  padding: 2rem;
+}
+
+/* Minimum setting to use IBM Plex font */
+@font-face {
+  font-weight: 400;
+  font-family: 'IBM Plex Sans';
+  font-style: normal;
+  src: local('IBM Plex Sans'), local('IBMPlexSans'),
+    url(https://fonts.gstatic.com/s/ibmplexsans/v6/zYXgKVElMYYaJe8bpLHnCwDKhdHeEw.woff) format('woff');

```

Comment:
Ah @IgnacioBecerra can we replace the google fonts here with the our own fonts?

```

/* Minimum setting to use IBM Plex font */
@font-face {
  font-weight: 400;
  font-family: 'IBM Plex Sans';
  font-style: normal;
  src: local('IBM Plex Sans'), local('IBMPlexSans'),
    url(https://1www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Sans/fonts/complete/woff/IBMPlexSans-Regular.woff) format('woff');
  font-display: auto;
}

@font-face {
  font-weight: 600;
  font-family: 'IBM Plex Sans';
  font-style: normal;
  src: local('IBM Plex Sans SemiBold'), local('IBMPlexSans-SemiBold'),
    url(https://1www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Sans/fonts/complete/woff/IBMPlexSans-SemiBold.woff) format('woff');
  font-display: auto;
}
```

Reply:
Thanks for pointing that out! I'm going to update the boilerplate script to add this 👍